src/ape/managers/converters.py

```diff
@@ -91,7 +92,52 @@ def convert(self, value: str) -> AddressType:
         return to_checksum_address(value)
 
 
-hex_address_converter = HexAddressConverter(None, None)  # type: ignore
+hex_address_converter = HexAddressConverter(None, None, None)  # type: ignore
+
+
+class ListTupleConverter(ConverterAPI):
+    """
+    A converter that converts all items in a tuple or list recursively.
+    """
+
+    def is_convertible(self, value: Any) -> bool:
+        return isinstance(value, (list, tuple))
+
+    def convert(self, value: Union[List, Tuple]) -> Union[List, Tuple]:
+        """
+        Convert the items inside the given list or tuple.
+
+        Args:
+            value (Union[List, Tuple]): The collection to convert.
+
+        Returns:
+            Union[list, tuple]: Depending on the input
+        """
+
+        converted_value: List[Any] = []
+
+        for v in value:
+            # Try all of them to see if one converts it over (only use first one)
+            conversion_found = False
+            # NOTE: Double loop required because we might not know the exact type of the inner
+            #       items. The UX of having to specify all inner items seemed poor as well.
+            for typ in self.converter._converters:
```

Comment:
I think we may need to filter out `list` and `tuple` here - that could explain why my lists are doubling.

Reply:
naw, fixed this in c7ddce6